app/mailers/user_mailer.rb

```diff
@@ -1,16 +1,21 @@
 class UserMailer < ActionMailer::Base
-  default from: "from@example.com"
+  default from: 'us@supermarkit.ca'
 
-  # Subject can be set in your I18n file at config/locales/en.yml
-  # with the following lookup:
-  #
-  #   en.user_mailer.reset_password_email.subject
-  #
   def reset_password_email(user)
-    raise
     @user = user
     @url  = edit_password_reset_url(user.reset_password_token)
-    mail(:to => user.email,
-         :subject => "Your password has been reset")
+    mail(:to => user.email, :subject => "Your password has been reset")
+  end
+
+  def activation_needed_email(user)
+    @user = user
+    @url = activate_user_path(@user)
+    mail( to: @user.email, subject: 'Join Supermarkit' )
+  end
+
+  def activation_success_email(user)
+    @user = user
+    @url = activate_user_path(@user)
+    mail( to: @user.email, subject: 'Shop with Supermarkit' )
```

Comment:
Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.